README.md

```diff
@@ -1 +1,4 @@
-# opencraft-simple-theme
\ No newline at end of file
+# EDX Simple theme
```

Comment:
@giovannicimolin. `EDX` is not the correct way to spell the name. `edX` is. In this case it is better to use `Open edX simple theme` here and leave the repository name as-is.

Reply:
:+1: